cumulus/ansible/tasks/cluster.py

```diff
@@ -137,10 +137,10 @@ def provision_cluster(playbook, cluster, profile, secret_key, extra_vars,
 @command.task
 def start_cluster(launch_playbook, provision_playbook, cluster, profile,
                   secret_key, launch_extra_vars, provision_extra_vars,
-                  girder_token, log_write_url, master_name=None):
+                  girder_token, log_write_url):
 
```

Comment:
Note the removal of master_name will break our current taskflows, no big deal we just need to remember to update them when this branch lands.